### PR TITLE
Fix undelete reuses and dataservices

### PR DESCRIPTION
**Pull request by ThibaudDauce**

### CODE REVIEW

File: udata/tests/api/test_dataservices_api.py [hunk 1]
```diff
@@ -101,8 +101,29 @@ def test_dataservice_api_create(self):
         self.assertEqual(dataservice.base_api_url, "https://example.org")
         self.assertIsNotNone(dataservice.deleted_at)
 
-        # response = self.get(url_for('api.dataservice', dataservice=dataservice))
-        # self.assert410(response)
```
Comment (maudetes) [inline udata/tests/api/test_dataservices_api.py:105]: I don't see where you've modified this in this PR?

**Review by maudetes — APPROVED** [review]
Thank you for the fix! :pray:

File: udata/tests/api/test_dataservices_api.py (hunk 1 above)
Comment (ThibaudDauce) [reply udata/tests/api/test_dataservices_api.py:105]: It was already working (only with a new user logged in, so I added it back)

Comment (maudetes) [reply udata/tests/api/test_dataservices_api.py:105]: Oh my bad, I hadn't see it was commented! Thank you :pray:

**Pull request merged**